tests/test_export_sample.py

```diff
@@ -22,32 +22,43 @@ def test_sample_udf_withudf(mocker, capsys):
 
 def test_sample_udf_withoutudf(mocker, capsys):
     # Test output for sample with no known udf in database
-    samples_mock[sample_name] = MyMock({"Dx geslacht": "Vrouw"})
+    samples_mock[sample_name] = [SampleMock({})]
     patched_clarity_epp = mocker.patch(
-        'clarity_epp.export.sample.get_samples',
+        'clarity_epp.export.sample.get_artifact_samples',
         return_value=samples_mock
     )
-    sample.sample_udf_dx("lims", sys.stdout, udf="udf2", column_name=column_name)
+    sample.sample_udf_dx("lims", sys.stdout, udf="Dx geslacht", column_name=column_name)
     captured = capsys.readouterr()
     assert captured.out == f"Sample\t{column_name}\n{sample_name}\tunknown\n"
 
-def test_sample_udf_withoutdxudf(mocker, capsys):
-    # Test output for sample with known udf in database, but not Dx-udf
-    samples_mock[sample_name] = MyMock({"Geslacht": "Vrouw"})
+def test_sample_udf_with_multiple_equal_values(mocker, capsys):
+    # Test output for sample with no known udf in database
+    samples_mock[sample_name] = [SampleMock({"Dx geslacht": "Vrouw"}), SampleMock({"Dx geslacht": "Vrouw"})]
     patched_clarity_epp = mocker.patch(
-        'clarity_epp.export.sample.get_samples',
+        'clarity_epp.export.sample.get_artifact_samples',
         return_value=samples_mock
     )
-    sample.sample_udf_dx("lims", sys.stdout, udf="Geslacht", column_name=column_name)
+    sample.sample_udf_dx("lims", sys.stdout, udf="Dx geslacht", column_name=column_name)
     captured = capsys.readouterr()
-    assert captured.out == f"Sample\t{column_name}\nWarning, udf is not type \'Dx\'\n"
+    assert captured.out == f"Sample\t{column_name}\n{sample_name}\tVrouw\n"
+
 
+def test_sample_udf_with_multiple_different_values(mocker, capsys):
```

Comment:
Nice addition to the test. 
However, it seems that the hotfix is mainly focused on sample with a known UDF and without a known udf.

Wouldn't it make sense to also include a test_sample_udf_with_multiple_with_missing_values? 
e.g. 
`samples_mock[sample_name]` = [SampleMock({"Dx geslacht": "Vrouw"}), `SampleMock({})]`

Reply:
Good suggestion, done.